admin-ui/src/component/my-units/UnitReservations.tsx

```diff
@@ -112,8 +112,8 @@ const merge = (
           .map((reservation) => ({
             title: reservation.name as string,
             event: reservation,
-            start: reservation.begin,
-            end: reservation.end,
+            start: new Date(reservation.begin),
+            end: new Date(reservation.end),
```

Comment:
(Might) need to check truthiness.

Reply:
There should be no situation where this is empty